README.md

```diff
@@ -53,24 +54,21 @@ called `MyStage` that includes CDK stacks for your app and you want to deploy it
 to two AWS environments (`BETA_ENV` and `PROD_ENV`):
 
 ```ts
-import { App } from 'aws-cdk-lib';
-import { ShellStep } from 'aws-cdk-lib/pipelines';
-import { GitHubWorkflow } from 'cdk-pipelines-github';
+import { App } from "aws-cdk-lib";
+import { ShellStep } from "aws-cdk-lib/pipelines";
+import { GitHubWorkflow } from "cdk-pipelines-github";
```

Comment:
My apologies, these changes were done by my VSCode running the eslint rules against the ts code blocks in the Markdown automatically when I saved. I can back these changes out if you prefer, just let me know.